main.go

```diff
@@ -77,3 +78,11 @@ func getEnvDefault(key string, defaultVal string) string {
 
 	return val
 }
+
+func getURLUsingPlek(key string) string {
+	url, err := plek.Find(key)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return url.String()
```

Comment:
@alext: yes, i was expecting that to be discussed during review of this PR. plek.rb returns strings, and we've been happy with that. so we should be good to return strings from plek.go as well. i'll work on that.